.github/workflows/e2e-test.yml

```diff
@@ -0,0 +1,38 @@
+name: Run TS-InSpec-Objects E2E Tests
```

Comment:
A general note for workflows: 
Should we add a workflow for publishing the package to npm?
Was the npm package that is there now published manually off of this branch? It seems like these updates are a part of that package.
https://www.npmjs.com/package/@mitre/inspec-objects